### PR TITLE
Committing profiles adding to Spring and Queue URL

### PR DESCRIPTION
**Pull request by wzacariotto-ebsco**

### CODE REVIEW

File: src/main/resources/application.yml [hunk 1]
```diff
@@ -1,26 +1,39 @@
...
-  port: ${service.port:8080}
+  port: ${service.port:8888}
 
 mongoDB:
   database: bookDB
   host: 127.0.0.1
   port: 27017
   
-sqs:
-  queueUrl: https://sqs.us-east-2.amazonaws.com/404669482207/book-track.fifo
```
Comment (lmontanari-ebsco) [inline src/main/resources/application.yml:17]: I believe this is Diego's URL, you should keep it when it isn't a test.

File: src/main/resources/application.yml [hunk 2]
```diff
@@ -1,26 +1,39 @@
...
+---
+spring:
+  profiles: test
+  
+sqs:
+  queueUrl: http://localhost:4576/queue/Megalodon-Queue
+
+---
+spring:
+  profiles: devqa, live, integration
```
Comment (lmontanari-ebsco) [inline src/main/resources/application.yml]: We should have only two profiles for this app, "test" and "non-test". We are not going to deploy this app to eureka. In other words, when it is not a "test" consider the "default" option.

Comment (lmontanari-ebsco) [inline src/main/resources/application.yml]: We should have only two profiles for this app, "test" and "non-test". We are not going to deploy this app to eureka. In other words, when it is not a "test" consider the "default" profile.

**Review by lmontanari-ebsco — CHANGES_REQUESTED** [review]

File: src/main/resources/application.yml (hunk 1 above)
Comment (wzacariotto-ebsco) [reply src/main/resources/application.yml:17]: Actually, as Darren mentioned we are not going to use Fifo Queue, so we need to update it anyway

File: src/main/resources/application.yml (hunk 2 above)
Comment (wzacariotto-ebsco) [reply src/main/resources/application.yml]: Ok!

File: src/main/resources/application.yml (hunk 1 above)
Comment (lmontanari-ebsco) [reply src/main/resources/application.yml:17]: OK, But leave that to another specific commit. But I don't think that we will change it.

**Pull request closed without merging**